tools/typelib-clang-tlb-importer/CMakeLists.txt

```diff
@@ -1,116 +1,101 @@
-cmake_minimum_required(VERSION 2.8)
+include(LLVMTools)
 
-# prevent loud warnings from gcc-4.9 caused by a macro-expansion the
-# AST-headers... this only clutters up the compiler output with nonsense
-add_definitions("-Wno-unused-local-typedefs")
+# we tested and developed _only_ against llvm-3.4
+llvm_find_config(3.4)
 
-# when compiling inside the rock-framework "Wall" is used. This adds alotta
-# more bogus warnings from Debians llvm-3.4 headers...
-add_definitions("-Wno-strict-aliasing")
+if (NOT LLVM_CONFIG_EXECUTABLE)
+    message(FATAL_ERROR "typelib-clang-tlb-importer: llvm-config was not found, cannot continue.")
+else()
```

Comment:
Would you mind if I remove the else() block completely ? (for the sake of simplicity ?).  Since we fail now if llvm is not present, we really don't need to have if/else clauses.